database/migrations/000102_properties_upstream_id_index.down.sql

```diff
@@ -0,0 +1,19 @@
+-- Copyright 2023 Stacklok, Inc
```

Comment:
2024

Reply:
sorry, I missed that. I will fix the year in another PR, I need to add better test coverage for some parts anyway